api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -1732,6 +1732,16 @@ private void enableMCHForms() {
                             visitDoesNotHaveEncounterOfType(EncounterTypes.ANC_FOLLOWUP),
                             and(patientIsFemale()))));
 
+            extensions.add(visitAction(Extensions.MCH_DELIVERY_VISIT_ACTION,            //TODO: working on this
+                    "ui.i18n.EncounterType.name." + EncounterTypes.MCH_DELIVERY,
```

Comment:
I think you want to add the uuid here (ie. EncounterTypes.MCH_DELIVERY.uuid() ).

Since this is only for Liberia, feel free to add the delivery form @BranfordTGbieor

Reply:
My bad! Thanks @lnball